src/index.js

```diff
@@ -1,34 +1,23 @@
 import './styles/Sass/main.sass';
+import { addScore } from './modules/api.js';
+import { user, score, renderList } from './modules/displayList.js';
 
-const body = document.querySelector('body');
+// const body = document.querySelector('body');
+const submitBtn = document.getElementById('submit');
+// const gameUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/9W40RRKEzdVGseFky5fH/scores/';
 
-const layout = () => {
-  body.innerHTML = `
-    <header><h1>Leaderboard</h1></header>
-    <main>
-      <section>
-        <div class="Side">
-          <div class="fstSect">
-            <h3>Recent Scores</h3>
-            <button class="btn">Refresh</button>
-          </div>
-          <ul class="recentScores">
-            <li>Name:96</li>
-            <li>Name: 87</li>
-            <li>Name:100</li>
-          </ul>
-        </div>
-        <div class="Side">
-          <h3>Add your score</h3>
-          <input type="text" /><input type="text" />
-          <button class="btn">Submit</button>
-        </div>
-      </section>
-    </main>
-    <footer>
-      <h3 class="copyRights">Copy rights reserved to <a href="https://github.com/GeekyHacks">geekyhacks</a></h3>
-    </footer>
-    `;
-};
+window.addEventListener('load', () => {
+  renderList();
+});
+// this will add userScore to the APi
+submitBtn.addEventListener('click', async (event) => {
+  event.preventDefault();
+  await addScore(user.value, score.value);
+});
 
-layout();
+// will use it next project
+// const refresh = document.getElementById('refresh');
+
```

Comment:
- [ ] The refresh functionality is a [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Implement%20the%20%22Refresh%22%20button%20that%20will%20get%20all%20scores%20for%20a%20game%20created%20by%20you%20from%20the%20API%20(receiving%20data%20from%20the%20API%20and%20parsing%20the%20JSON).) for this milestone but  you have it commented here. Please I kindly suggest you add this functionality to your project and resubmit 👍